### DISCUSSION TITLE
which meditations should I try next?

### DISCUSSION THREAD

**Discussion by shane0** [shane0/shane0.github.io#37] [Q&A] [answered]
- research & experiment with meditations I have not tried yet

**Comment by shane0** [comment]
# Satipatthana

- https://www.dhammatalks.org/suttas/MN/MN10.html

Characteristic | Satipatthana | Anapanasati | Sutta
-- | -- | -- | --
Focus | The body, feelings, mind, and phenomena | The breath | Satipatthana Sutta
Goal | Develop mindfulness of all aspects of experience | Develop concentration, focus, and calmness | Anapanasati Sutta
Benefits | Increased mindfulness, reduced stress and anxiety, improved concentration and focus, increased self-awareness, enhanced emotional intelligence, greater compassion and kindness, a more peaceful and joyful life | Increased concentration, focus, and calmness

**Comment by shane0** [comment]
# asubha

- https://www.dhammatalks.org/suttas/AN/AN4_163.html

**Comment by shane0** [comment]
# maranasati

https://youtu.be/ctKYkBHTEaM

**Comment by JoyL8896** [comment]
Silva ultramind by Mindvalley?

**Comment by shane0** [comment, marked as answer]
linji chan
